components/locale-chooser.js

```diff
@@ -85,16 +88,31 @@ class LocaleChooser extends React.Component {
     open: false,
   }
 
-  handleClick() {
+  componentDidMount() {
+    document.addEventListener('click', this.clickOutside.bind(this), true)
```

Comment:
This is a bit misleading since `clickOutside` is fired even when clicking _inside_ the dropdown.